cpp/tests/reductions/list_rank_test.cpp

```diff
@@ -131,7 +131,7 @@ TEST_F(ListRankScanTest, ListOfStruct)
      false,
      false}};
   auto col2 = cudf::test::strings_column_wrapper{
-    {"x", "x", "a", "a", "b", "b", "a", "b", "a", "b", "a", "c", "a", "c", "a", "c", "b", "b"},
+    {"x", "x", "a", "a", "b", "", "a", "b", "a", "b", "a", "c", "a", "c", "", "", "b", "b"},
```

Comment:
nit: I typically have a `constexpr auto null = "";`, and use this `null` in initializer list. (to differentiate between empty string and nulls.